src/store/index.js

```diff
@@ -50,15 +52,28 @@ export default () => {
             },
             updateWaitingForInitialSearch(state, value) {
                 state.waitingForInitialSearch = value;
+            },
+            updateUrl(state) {
```

Comment:
This is not a mutation. Migrate it somewhere more appropriate (e.g. a function in QueryString.js)